### ISSUE TITLE
Bug in TTXN again (probably because of changes in TTX (FontTools)

### ISSUE THREAD

**Issue by Arno-Enslin** [adobe-type-tools/afdko#115]
There is a bug in ttxn.py again. It doesn't work with the latest FontTools. ttxn.py was fixed in January 2016. And if I replace ttxn.py in the official AFDKO 2.5 64958, ttxn from January 2016 works (more or less). (TTXN contained in AFDKO 2.5 64958 does not work.) But in the branch of the AFDKO and with the FontTools (both from 03-26-2016) ttxn does not work anymore. So either the "bug" has come back or it is a new "bug". Well, it seems to be more an incompatibility with FontTools.

I think, TTXN should be integrated in FontTools but not in the AFDKO, because I assume, that this reduces the risk for these incompatibilities.

**Comment by anthrotype** [comment]
Could you provide more information as to what exactly is no longer working  in ttxn? E.g. the error message with the full traceback?

I don't think `ttxn.py` could be integrated in FontTools as it is, since it depends on at least on another AFDKO tool (`tx`) to decompile some of the font data.

We are planning to add support in `fontTools.feaLib` for decompiling binary OpenType Layout tables back to FEA code (as well as other OTL source formats), which is what ttxn.py does.
However, we first want to complete the compilation part, which is now almost done (thanks to @brawer's great work).

**Comment by Arno-Enslin** [comment]
> We are planning to add support in fontTools.feaLib for decompiling binary OpenType Layout tables back to FEA code (as well as other OTL source formats)

Very cool!

---------------------
My batch:

```
@echo off
call FDKvars
for %%f in (Decompile\*.otf; Decompile\*.ttf Decompile\*.woff; Decompile\*.woff2) do %TTXNcom% -se -t GPOS -d DecompileD "%%f"
```
TTXNcom in FDKvars is: C:\FDK_26.03.2016\Tools\win\ttxn

And the error:

```
  File "C:\FDK_26.03.2016\Tools\win\Python\AFDKOPython27\lib\site-packages\fontt
ools-3.0-py2.7.egg\fontTools\ttLib\__init__.py", line 302, in saveXML
    self._tableToXML(tableWriter, tag, progress)
**TypeError: _tableToXML() takes exactly 5 arguments (4 given)**
```

**Comment by anthrotype** [comment]
The `quiet` argument was deprecated with the introduction of the new logging system. The latter needs to be used instead to control the verbosity of fontTools. I will fix that in the same pull request #116.
